coral/src/app/layout/header/HeaderNavigation.tsx

```diff
@@ -57,6 +59,7 @@ function HeaderNavigation() {
               icon={notifications}
               linkText={"Go to approve requests"}
               href={Routes.APPROVALS}
+              showNotificationBadge={totalPendingRequests > 0}
```

Comment:
Not sure if there was a sync already with Venla, but I think it would be nice to have this implemented like it's in Angular currently:

<img width="447" alt="Screenshot 2023-11-30 at 20 15 10" src="https://github.com/Aiven-Open/klaw/assets/943800/326706b0-f9ca-4f86-bb11-8cb02cc7fa64">


That way, the user can see here they have approvals waiting and go directly to the link (e.g. direct link to acl approvals etc)

Reply:
Yes, this was planned for a subsequent PR in the "Other possible improvement" section of the issue ^^